common/wbdisable/index.json-ld

```diff
@@ -0,0 +1,35 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Basic HTML",
+		"fr": "HTML simplifiée"
+	},
+	"description": {
+		"en": "Enable Basic HTML Mode",
+		"fr": "Passer à la version HTML simplifiée"
+	},
+	"modified": "2023-07-04",
+	"componentName": "wb-disable",
```

Comment:
The component name need to match the folder name. Please rename the folder for "wb-disable"